src/components/control/multi-select.jsx

```diff
@@ -176,6 +176,7 @@ const MultiSelect = forwardRef(function MultiSelect(props, ref) {
 
   function onClick(event) {
     if (event.defaultPrevented) return;
+    if (event.target !== refs.autocomplete.current) refs.autocomplete.current.focus();
```

Comment:
I can't test this because you have to "click" the background div (which is not `screen.getBy`able). I am leaving this up to QA to verify this behavior.

Reply:
We could give it a `testID` or test it in Cypress, but tbh I think it's fine without an automated test.

Reply:
Yeah, I'm leaning towards only interacting with accessible element in tests. But that's just my "all-in" mentality. 😬